openpower/patches/p8dtu-patches/linux/0001-VGA-handshake-with-BMC.patch

```diff
@@ -0,0 +1,68 @@
+From 4c5f6ff7a2b7c2df17d16c3ce18b19036be95223 Mon Sep 17 00:00:00 2001
+From: Jim Yuan <jim.yuan@supermicro.com>
+Date: Mon, 17 Dec 2018 15:57:05 -0800
+Subject: [PATCH] VGA handshake with BMC
```

Comment:
The same comments as per https://github.com/open-power/op-build/pull/2711#discussion_r267586104 apply here.

Reply:
can we let this one go for now and come back to clean up.  We need to get p8dtu config upstream so we can do B&S build.

Reply:
Let's drop the patch.